sql/expression/arithmetic_test.go

```diff
@@ -146,13 +146,14 @@ func TestDiv(t *testing.T) {
 		name        string
 		left, right float64
 		expected    float64
+		null        bool
 	}{
-		{"1 / 1", 1, 1, 1},
-		{"-1 / 1", -1, 1, -1},
-		{"0 / 1234567890", 0, 12345677890, 0},
-		{"3.14159 / 3.0", 3.14159, 3.0, float64(3.14159) / float64(3.0)},
-		{"1/0", 1, 0, math.Inf(1)},
-		{"-1/0", -1, 0, math.Inf(-1)},
+		{"1 / 1", 1, 1, 1, false},
+		{"-1 / 1", -1, 1, -1, false},
+		{"0 / 1234567890", 0, 12345677890, 0, false},
+		{"3.14159 / 3.0", 3.14159, 3.0, float64(3.14159) / float64(3.0), false},
+		{"1/0", 1, 0, math.Inf(1), true},
```

Comment:
These are wrong

Reply:
I changed the expected val, but it's ignored if null is true